acceptance-tests/mysql_test.go

```diff
@@ -13,15 +18,30 @@ import (
 	"csbbrokerpakgcp/acceptance-tests/helpers/services"
 )
 
+type AppResponseUser struct {
```

Comment:
A bit pedantic (and non-blocking for merge), but the type should start with a lower case as we are not consuming it outside of the package. However the members do need a leading capital otherwise `encoding.JSON` can't see them.